packages/swirl-components/src/components/swirl-tag/swirl-tag.tsx

```diff
@@ -26,15 +30,42 @@ export type SwirlTagSize = "s" | "m";
 export class SwirlTag {
   @Element() el: HTMLElement;
 
+  @Prop() icon?: string;
   @Prop() intent?: SwirlTagIntent = "default";
   @Prop() label!: string;
   @Prop() removable?: boolean;
   @Prop() bordered?: boolean;
   @Prop() size?: SwirlTagSize = "m";
   @Prop() removalButtonLabel?: string = "Remove";
+  @Prop({ mutable: true }) variant?: SwirlTagVariant = "default";
 
   @Event() remove?: EventEmitter<MouseEvent>;
 
+  private iconEl: HTMLElement;
+
+  componentDidLoad() {
+    this.forceIconProps();
+  }
+
+  componentWillLoad() {
+    this.forceVariant();
+  }
+
+  private forceIconProps() {
+    const icon = this.iconEl?.children[0];
+
+    icon?.setAttribute("size", "16");
+  }
+
+  private forceVariant() {
+    if (Boolean(this.bordered)) {
```

Comment:
To keep this change backwards compatible, I implemented this fallback of the `bordered` prop to the new outline `variant`.